libraries/AP_DDS/AP_DDS_Client.h

```diff
@@ -29,9 +36,20 @@ class AP_DDS_Client
 
 private:
 
+    AP_Int8 enabled;
+
+#if AP_DDS_UDP_ENABLED
+    AP_Int32 udp_port;
+    // UDP endpoint
+    const char* ip = "127.0.0.1";
+    // UDP Allocation
+    uxrUDPTransport udp_transport;
+#endif // AP_DDS_UDP_ENABLED
+
     // Serial Allocation
     uxrSerialTransport serial_transport; // client uxr serial transport
     uxrSession session; //Session
+    bool is_serial;
```

Comment:
Please document this flag. Perhaps rename to `is_using_serial`?

Reply:
done